content/cli/orch_cli/plugins.md

```diff
@@ -280,6 +280,72 @@ Plugin `e90b1a09-6b56-4a92-b9cd-5fc4ef32ab74` is now owned by user `admin`.
 ...
 {{< /highlight >}}
 
+
+### blueprint-labels
+
+#### blueprint-labels list
+`cfy plug blueprint-labels list [OPTIONS] PLUGIN_ID`
+
+List blueprint labels for specific plugin.
+
+`PLUGIN_ID` is the id of the plugin.
+
+#### blueprint-labels add
+`cfy plug blueprint-labels add [OPTIONS] LABELS_LIST PLUGIN_ID`
+
+Add blueprint labels for specific plugin.
+
+`PLUGIN_ID` is the id of the plugin.
+
+`LABELS_LIST` is a list in `<label1>:<value1>,<label2>:<value2>` format of labels.  Any comma and
+colon in `<value>` must be escaped with `\`.
```

Comment:
And backslash values?

Reply:
They work fine, e.g. `cfy plug blueprint-labels add foo:/ ce5562c6-6e35-49ae-a74e-4f3e0b98e63c` will produce

```
Listing blueprint_labels of plugin ce5562c6-6e35-49ae-a74e-4f3e0b98e63c...

Plugin labels
+---------------+----------------------------------+
|      key      |              values              |
+---------------+----------------------------------+
|      foo      |               "/"                |
+---------------+----------------------------------+
```

Reply:
That's a forwardslash. What happens if I want foo:\

Reply:
`cfy plug blueprint-labels add bar:\\ ce5562c6-6e35-49ae-a74e-4f3e0b98e63c`

```
Listing blueprint_labels of plugin ce5562c6-6e35-49ae-a74e-4f3e0b98e63c...

Plugin labels
+---------------+----------------------------------+
|      key      |              values              |
+---------------+----------------------------------+
|      foo      |               "/"                |
|      bar      |               "\"                |
+---------------+----------------------------------+
```